docs/Tutorial.rst

```diff
@@ -9,11 +9,13 @@ You can setup and use your own endpoint by following the `endpoint documentation
 funcX Python SDK
 ----------------
 
-The funcX Python SDK provides programming abstractions for interacting with the funcX service. Before running this tutorial you should first install the funcX SDK as follows:
+The funcX Python SDK provides programming abstractions for interacting with the funcX service. Before running this tutorial you should first install the funcX SDK in its own `venv <https://docs.python.org/3/tutorial/venv.html>`_ environment:
```

Comment:
Should the `_` be here after the closing backtick?

Reply:
Isn't the _ required for links?  https://stackoverflow.com/questions/51858956/what-is-the-standard-syntax-for-hyperlink-in-restructuredtext-foo-or-foo

Reply:
Looks like it 👍